security/wildfly-mods/build_xml.patch

```diff
@@ -0,0 +1,65 @@
+--- /home/darranl/jakartaee10/wildfly-tck-runners/authentication/authentication-tck/bin/build.xml	2022-08-04 22:11:52.000000000 +0100
++++ /home/darranl/jakartaee10/authentication/tck/old-tck/source/install/jaspic/bin/build.xml	2022-08-04 17:53:23.918558141 +0100
```

Comment:
@darranl Was this file added by mistake?

Reply:
FYI for future reference this file will be reworked further as I active the old-tck job.

Reply:
Answering myself -- no, it was not!